core/src/miner/mem_pool_types.rs

```diff
@@ -429,22 +429,22 @@ pub struct AccountDetails {
 
 #[derive(Default, Clone, Copy, Debug, PartialEq)]
 /// Minimum fee thresholds defined not by network but by Mempool
-pub struct MemPoolFees {
-    min_pay_transaction_cost: u64,
-    min_set_regular_key_transaction_cost: u64,
-    min_create_shard_transaction_cost: u64,
-    min_set_shard_owners_transaction_cost: u64,
-    min_set_shard_users_transaction_cost: u64,
-    min_wrap_ccc_transaction_cost: u64,
-    min_custom_transaction_cost: u64,
-    min_asset_mint_cost: u64,
-    min_asset_transfer_cost: u64,
-    min_asset_scheme_change_cost: u64,
-    min_asset_supply_increase_cost: u64,
-    min_asset_unwrap_ccc_cost: u64,
+pub struct MemPoolMinFees {
```

Comment:
@Byeongjee Can you separate this commit into the "Renaming MemPoolFees" commit and the "Creating an RPC" commit? If it is hard, it's okay to maintain the commit since this commit is small.

It is a good practice to create a commit for refactoring. When we review a commit, we focus on the different aspects when reviewing a refactoring commit and feature commit.
When we review a refactoring commit, we focus on that the behavior is changed or not changed. A refactoring commit should not change behavior. When we review a feature commit, we focus on whether the code is changed as expected.